source/connectors/kafka-sink-postprocessors.txt

```diff
@@ -9,10 +9,821 @@ Kafka Sink Connector Post-Processors
+   }
+
+When the connector processes sink data that matches an existing document with
+the same business key field values, it updates the document with the new
+values without changing the ``_id`` field.
+
+.. code-block:: json
+   :emphasize-lines: 7
+
+   {
+     "_id": ObjectId('5db0b81199b7fbcc89631d06'),
+     "street": "50th Street",
+     "avenue": "Broadway",
+     "parking_spaces": {
+       "capacity": 407,
+       "occupied": 402
+     }
+   }
+
+.. _writemodel-strategy-timestamps:
+
+WriteModel Strategy: Inserted and Modified Timestamps
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This example shows how we can track the create and update timestamps on
+documents that are upserted by the connector. The
+``UpdateOneTimestampsStrategy`` custom write model strategy performs the
+following tasks:
+
+* When a new MongoDB document is inserted into the collection by the
+  connector, the ``_insertedTS`` and ``_modifiedTS`` fields will be set to
+  the current time.
+
+* When an existing MongoDB document is updated in the collection by the
+  connector, the ``_modifiedTS`` field will be updated to the current
+  time.
+
+This is set by specifying the ``UpdateOneTimestampsStrategy`` in the
+configuration file as follows:
+
+.. code-block:: properties
+
+   writemodel.strategy=com.mongodb.kafka.connect.sink.writemodel.strategy.UpdateOneTimestampsStrategy
+
+For this example, we want to track the position of a train along its
+route. The ``_insertedTS`` field provides us with the time and date that the
+first position report was saved into the collection. The ``_modifiedTS`` field
+provides us with the time and date that the most recent report was saved
+into the collection.
+
+The position report includes the following data in the value document:
+
+.. code-block:: json
+
+   {
+     "_id": "MN-1234",
+     "start": "Beacon",
+     "destination": "Grand Central"
+     "position": [ 40.8051693, -73.9388079 ]
+   }
+
+The writemodel strategy is set to ``UpdateOneTimestampsStrategy`` to
+append the created and modified timestamps and the document id strategy is
+set to ``ProvidedInValueStrategy`` to identify the train using the ``_id``
+field of the value document.
+
+.. code-block:: properties
+
+   document.id.strategy=com.mongodb.kafka.connect.sink.processor.id.strategy.ProvidedInValueStrategy
+   writemodel.strategy=com.mongodb.kafka.connect.sink.writemodel.strategy.UpdateOneTimestampsStrategy
+
+The MongoDB document inserted after processing the initial message for the
+train includes the following data:
+
+.. code-block:: json
+   :emphasize-lines: 3-4
+
+   {
+     "_id": "MN-1234",
+     "_insertedTS": ISODate('2019-10-23T15:08:000Z"),
+     "_modifiedTS": ISODate("2019-10-23T15:08:000Z"),
+     "start": "Beacon",
+     "destination": "Grand Central"
+     "position": [ 40.805, -73.939 ]
+   }
+
+After an hour, the train reports its current location along its route. The
+``position`` and ``_insertedTS`` fields are updated:
```

Comment:
shouldn't this be `_modifiedTS`?

Reply:
Good catch, will fix.

Reply:
Still needs changed.

Reply:
Fixed in latest commit 11267f48f8ae578399a2902334b14db2bbb36a8d